doc/reference/configuration/configuration_reference.rst

```diff
@@ -11,6 +11,329 @@ This topic describes all :ref:`configuration parameters <configuration>` provide
+
+    **Example**
+
+    In the example, only the events of ``bands`` and ``singers`` spaces are logged:
+
+    ..  code-block:: yaml
+
+        audit_log:
+          spaces: [bands, singers]
+
+    |
+    | Type: array
+    | Default: box.NULL
+    | Environment variable: TT_AUDIT_LOG_SPACES
+
+..  _configuration_reference_audit_syslog-facility:
+
+..  confval:: audit_log.syslog_facility
+
+    Specify a system logger keyword that tells `syslogd <https://datatracker.ietf.org/doc/html/rfc5424>`__ where to send the message.
+    You can enable logging to a system logger using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+
+    See also: :ref:`syslog configuration example <configuration_reference_audit_syslog-example>`.
+
+    |
+    | Type: string
+    | Possible values: 'auth', 'authpriv', 'cron', 'daemon', 'ftp', 'kern', 'lpr', 'mail', 'news', 'security', 'syslog', 'user', 'uucp', 'local0', 'local1', 'local2', 'local3', 'local4', 'local5', 'local6', 'local7'
+    | Default: 'local7'
+    | Environment variable: TT_AUDIT_LOG_SYSLOG_FACILITY
+
+..  _configuration_reference_audit_syslog-identity:
+
+..  confval:: audit_log.syslog_identity
+
+    Specify an arbitrary string that will be placed at the beginning of all messages.
+    You can enable logging to a system logger using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+
+    See also: :ref:`syslog configuration example <configuration_reference_audit_syslog-example>`.
+
+    |
+    | Type: string
+    | Default: 'tarantool'
+    | Environment variable: TT_AUDIT_LOG_SYSLOG_IDENTITY
+
+..  _configuration_reference_audit_syslog-server:
+
+..  confval:: audit_log.syslog_server
+
+    Set a location for the syslog server.
+    It can be a Unix socket path starting with 'unix:' or an ipv4 port number.
+    You can enable logging to a system logger using the :ref:`audit_log.to <configuration_reference_audit_to>` option.
+
+..  _configuration_reference_audit_syslog-example:
+
+    **Example**
+
+    ..  literalinclude:: /code_snippets/snippets/config/instances.enabled/audit_log_syslog/config.yaml
+        :language: yaml
+        :start-at: audit_log:
+        :end-at: 'tarantool'
+        :dedent:
+
+    -   :ref:`audit_log.syslog_server <configuration_reference_audit_syslog-server>` -- a syslog server location.
+
+    -   :ref:`audit_log.syslog_facility <configuration_reference_audit_syslog-facility>` -- a system logger keyword that tells syslogd where to send the message.
+        The default value is ``local7``.
+
+    -   :ref:`audit_log.syslog_identity <configuration_reference_audit_syslog-identity>` -- a string placed at the beginning of every message.
+        The default value is ``tarantool``.
+
+    These options are interpreted as a message for the `syslogd <https://datatracker.ietf.org/doc/html/rfc5424>`_ program,
+    which runs in the background of any Unix-like platform.
+
+    An example of a Tarantool audit log entry in the syslog:
```

Comment:
Hm. The example is JSON, not in the syslog format. But this sentence states that it is from syslog.

We can write JSON messages to syslog (AFAIR, they shouldn't contain newline symbols in this case) and they will be formatted with all the proper headers and then the JSON data on the same line.